vars/createGithubIssue.groovy

```diff
@@ -14,26 +14,46 @@
  @param args.issueBody <required> - GitHub issue body
  @param args.label <optional> - GitHub issue label to be attached along with 'untriaged'. Defaults to autocut.
  */
+
 void call(Map args = [:]) {
     label = args.label ?: 'autocut'
     try {
         withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
-            def issues = sh(
-                    script: "gh issue list --repo ${args.repoUrl} -S \"${args.issueTitle} in:title\" --label ${label}",
-                    returnStdout: true
-            )
-            if (issues) {
-                println('Issue already exists, adding a comment.')
-                def issuesNumber = sh(
+            def openIssue = sh(
                     script: "gh issue list --repo ${args.repoUrl} -S \"${args.issueTitle} in:title\" --label ${label} --json number --jq '.[0].number'",
                     returnStdout: true
-                ).trim()
+            ).trim()
+
+            def currentDayMinusThree = sh(
+                script: "date -d \"3 days ago\" +'%Y-%m-%d'",
```

Comment:
Do you want to take this as an argument? How many days ago do we want to re-open issues for? Maybe just take the integer as an arg with default value as 3?

Reply:
I thought so,  but again if we have to give that option to user, we should go with complete regex meaning
`daysToReOpen: 3 days ago` or `daysToReOpen: 6 hours ago` or `daysToReOpen: closed in last 2 months` or even more. This should be compatible with linux `date` and with github search query (converting back to yyyy-MM-dd format). Just giving an option to use to change in days might not be very useful.

Reply:
We can just start with integer value fixed to number of days. WDYT? I am assuming we will soon need to change this too so avoiding another release just to fix that 1 value

Reply:
Ya I can have it hardcoded in days, but the number of days user can configure through the library. Will make that change soon.

Reply:
@gaiksaya I have updated the code in my latest commit, also update the description with testing information.